othello/moderator/constants.py

```diff
@@ -0,0 +1,72 @@
+import enum
+
+
+class Player(enum.Enum):
+    BLACK = "x"
+    WHITE = "o"
+    EMPTY = "."
+
+    @classmethod
+    def opposite_player(cls, current_player):
+        return cls.BLACK if current_player == cls.WHITE else cls.WHITE
+
+    @classmethod
+    def to_legacy(cls, player):
+        return "@" if player == cls.BLACK else cls(player).value
+
+    @classmethod
+    def from_legacy(cls, player):
+        return cls.BLACK.value if player == "@" else cls(player).value
```

Comment:
```suggestion
        return cls.BLACK.value if player == "@" else cls(player)
```

Reply:
The two return values are not the same type

Reply:
Updated.

Reply:
There is actually no use for this method

Reply:
Hmm. Well, it's there if the protocol changes in the future.